src/panes.ts

```diff
@@ -82,6 +82,21 @@ export type SearchAndSelectPane = Pane<
   { value: string | string[] }
 >
 
+export type BrandSelectPane = Pane<
+  "brand_select_pane",
+  {
+    options: Array<{
+      label: string
+      sublabel?: string
+      value: string
+      image_url?: string
```

Comment:
🐧 I think this property is required on Seam Connect so we could make it required here too

Reply:
Ah, good to know. Shall I make it required on `SearchAndSelectPane` as well then?

Reply:
I'd probably leave `SearchAndSelectPane` as is for now since it's intended to be more generic